docs/samples/contact-center/app.js

```diff
@@ -143,6 +148,7 @@ function register() {
         teamsDropdown.innerHTML = ''; // Clear previously selected option on teamsDropdown
         const listTeams = agentProfile.teams;
         agentId = agentProfile.agentId;
+        agentAuxCodeId = agentProfile.defaultWrapupCode;
```

Comment:
This should be done using dropdown.

Reply:
Noted. As discussed offline, I'll be adding a dropdown here which is populated with the profile.wrapupcodes. Thanks for the ctach!